test/integration/targets/btrfs_quota/aliases

```diff
@@ -0,0 +1,6 @@
+needs/privileged
+needs/root
+shippable/posix/group1
+skip/rhel/8.0/3
```

Comment:
```suggestion
skip/rhel8.0
```

Reply:
Fixing this will allow the tests to run on RHEL 7.6.

Since the tests are marked `needs/privileged` they will not run on any of our Docker containers.